packages/server/src/utilities/schema.ts

```diff
@@ -155,3 +188,43 @@ export function parse(rows: Rows, schema: Schema): Rows {
     return parsedRow
   })
 }
+
+function isValidBBReference(
```

Comment:
Can this be simplified, to just:
```js
const isValid = (userIdArray: string[]) => {
  const found = userIdArray.find(userId => !db.isGlobalUserID(d._id))
  return !found
}
switch (columnSubType) {
  case FieldSubtype.USER:
    if (typeof columnData !== "string") {
      return false
    }
    const userIdArray = parseCsvExport(columnData)
    if (userIdArray.length > 1) {
      return false
    }
    return isValid(userIdArray)
  case FieldSubtype.USERS:
    if (!Array.isArray(columnData)) {
      return false
    }
    return isValid(columnData)
}
```

Reply:
Both types require parsing the `columnData`, as it comes as CSV string